### DISCUSSION TITLE
v2.7.1

### DISCUSSION THREAD

**Discussion by Limraj** [SCADA-LTS/Scada-LTS#2283] [General]
## 

The changelist is listed below:

### 📄️ GitHub issues/pullrequest list overview:

2.7.1

O - backend/old views

- #1606 Integrate REST API with permissions
- #1623 Alarm Notification Flag that use WebSockets
- #1716 Custom CSS settings
- #1721 Support authentication in HTTP retriever
- #1722 Support JSON in HTTP publisher
- #1757 Generate Junit Test XML Reports
- #1765 Build Docker Image using GitHubActions
- #1789 Fixing #1782 issue with usage of different approach
- #1828 Adding a new logging type
- #1870 WebSockets for Vue.js User Interface pattern
- #1876 AmCharts Live update by WebSocket
- #1878 WebSocket AmCharts
- #1886 Refactor of getSubject method in EmailContentUtils
- #1921 Get highest alarm level for user
- #1929 Refactor JSONUser class which is going too deep into the application layers
- #1935 System Settings consistency
- #1936 Purging data based on values limit
- #1938 Graphics unplugged on graphical view
- #1957 Simple Point Table Graphical View component
- #1978 Fixed event list
- #1982 Update Log4j to increase security
- #1994 Secure DWR methods
- #2020 Import/Export for reports
- #2026 Fixed HttpRetriver migration
- #2031 Fixed aggregation api for multistate
- #2036 Old UI Embedded mode disable
- #2042 simplePoint changevalue popup has no background
- #2045 Server error on user update
- #2047 User not authorized error in new UI
- #2072 Version upgrade for Synoptic Panel migration
- #2089 Fix vulnerabilities reported in #1734
- #2093 Added an endpoint returning the hierarchy of datapoints
- #2130 Fixed ReportDAO.search
- #2132 Updated docker-compose.yml with CATALINA_OPTS
- #2137 Docker-compose wait for start database
- #2173 Fixed import project
- #2188 Http headers configuration in system settings 
- #2191 Corrected security for uploads images
- #2199 Corrected path env.properties
- #2255 Corrected schema version for spring-security
- #2261 Corrected schema version in web.xml
- #2277 Fixed background loading in public view
- #2279 Enabled async-supported

V - new views [vue]

- #1607 Integrate SVG Synoptic Panels with new User Interface
- #1641 DataSources as new Vue.js page
- #1762 Corrections for the Synoptic Panel
- #1859 Users as new Vue.js page
- #1860 User Profiles as new Vue.js page
- #1877 Event List as new Vue.js page
- #1879 WatchList as new Vue.js page
- #1933 Add navigation to Modern User Interface from OldUI Pages
- #1934 Complete new User Interface
- #1946 Point Hierarchy as Vue.js page
- #1974 Modern Interface navigation bar
- #1973 Graphical views v2.0 - Vue.js page 
- #2006 Selection of DataPoint to be rendered on chart (Vue.js WatchList)
- #2118 Taking into account the colour configuration for TextRenderer 
- #2167 Fixed graphical view component
- #2243 Disabled Graphical View 2.0
- #2245 Corrected sorting of folders and points on watchlist

ScadaBR.war file checksum: ``

<hr /><em>This discussion was created from the release <a href='https://github.com/SCADA-LTS/Scada-LTS/releases/tag/v2.7.1'>v2.7.1</a>.</em>

**Comment by nex2000** [comment]
### Impossible to change graphic line colour

In the new UI is not possible change line colour , the line remains unchanged GREEN

**Comment by nex2000** [comment]
![LTS1](https://user-images.githubusercontent.com/110027808/181212300-d034c981-d959-4904-a7f9-f022e24ce8d8.jpg)

The user and password work fine in old UI

**Comment by Limraj** [comment]
Hello,
login from the new ui will be improved in the next version 2.7.1.1. 

However, as for the problem with colors, I encourage you to create an issue with the scenario, I think there is a chance that in the upcoming versions, we will introduce a correction. I also encourage you to report further bugs. We appreciate your commitment.

Best wishes.
